src/app/features/collection/components/progress-bar-skeleton/progress-bar-skeleton.component.html

```diff
@@ -0,0 +1,7 @@
+<div class="tw-flex tw-flex-col tw-items-center tw-w-full tw-p-1 tw-bg-zinc-50 tw-text-xs tw-text-black">
+  <p>{{ label() | translate }}</p>
+  <div class="tw-flex tw-items-center tw-gap-1 tw-w-full">
+    <ion-skeleton-text animated style="width: 10%;" class="custom-skeleton tw-mr-2"/>
```

Comment:
it will be nice if we can aboid inline styles in favor of class styling via tailwind, to keeep same style of code. If we need to prioritize style - use "!" before tailwind class, like '!tw-w-1/10". We also can add new property to tailwind settings if they do not provide it, so 10% can be used then as buid in property https://tailwindcss.com/docs/configuration

Reply:
also it will be nice to use more specific class name, like 'progress-bar-skeleton' if we will need that class afterward. It will give us more precise results in global search of the codebase.